test/unit/flux/t0001-flux.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+
+from ats.tests import AtsTest
+
+class TestFluxScheduled(unittest.TestCase):
+    @classmethod
+    def setUp():
```

Comment:
If it's a classmethod it needs to take ``cls`` as the first argument. But ``setUp`` is an instance method and so should take ``self`` as the first argument, see the example here https://docs.python.org/3/library/unittest.html#unittest.IsolatedAsyncioTestCase.run

But also I might not use ``setUp`` at all and might create an ``AtsTest()`` instance in each test function. That's up to you though, depends on how much duplication it would create.

Reply:
I think I'm leaning towards creating an `AtsTest` in each function. That way, I could test both valid and invalid ones.